addons/crm_livechat/models/mail_channel.py

```diff
@@ -28,7 +28,7 @@ def _convert_visitor_to_lead(self, partner, key):
         :param key: operator input in chat ('/lead Lead about Product')
         """
         description = ''.join(
-            '%s: %s\n' % (message.author_id.name or self.anonymous_name, message.body)
+            '%s: %s<br/>' % (message.author_id.name or self.anonymous_name, html2plaintext(message.body))
```

Comment:
I have to convert it in plaintext because of formating. As due to  <p> tag in message body it is printing in new line and we are not getting desired output in description field of crm.

Reply:
Done